packages/netsuite-adapter/src/client/suiteapp_client/suiteapp_client.ts

```diff
@@ -359,12 +359,19 @@ export default class SuiteAppClient {
       )
       return
     }
-    const { results } = result
-    const featureFields = results[0].data.fields
-    const requiredFeatures = [SDF, SUITE_BUNDLER, SOAP, REST, TBA]
-    const featuresToEnable = requiredFeatures.filter(feature => featureFields[feature] === 'F')
+    const featureFields = result.results?.[0]?.data?.fields
+    if (typeof featureFields === undefined) {
+      log.error('Recieved an unexpected response from the restlet request')
+      return
+    }
+    const unexpectedValueFeatures = REQUIRED_FEATUERS.filter(feature => !['F', 'T'].includes(featureFields[feature] as string))
+    if (unexpectedValueFeatures.length !== 0) {
+      const featureValuePair = unexpectedValueFeatures.map(e => `${e} - ${featureFields[e]}`)
+      log.error('The following features have the following unexpected values: ', featureValuePair.join(', '))
```

Comment:
The `toString()` value of  plain object is `[object Object]`. I think it would be better to log it as an object and not as string:
```suggestion
      log.error('The following features have the following unexpected values: %o', unexpectedValueFeatures.map(e => ({ [e]: featureFields[e] })))
```